gapis/gfxapi/resource.go

```diff
@@ -25,8 +25,11 @@ type ResourceMap map[Resource]id.ID
 
 // Resource represents an asset in a capture.
 type Resource interface {
-	// ResourceName returns the UI name for the resource.
-	ResourceName() string
+	// ResourceName returns the UI identity for the resource.
```

Comment:
Here's my thinking:
1. Although the path is unique, there is no value in displaying it to the user.
2. We should display the GL texture/shader/program names. There are many places where we only show the GL names, so I don't think we can get away with not allowing the user to use the GL name to lookup a texture in the texture view, for example. Even if we change the linking, a user will inevitably find themselves in a situation where they want to find "texture 5". I know that the names are not unique, though for the "interesting" resources they typically are unique within a resource type.
3. Separating the label from the name and letting the UI do the formatting is preferable, 1st normal form and all.
4. I changed the name to be numeric to allow sorting. I don't think we'll run into an API that won't use a numeric identifier for resources. I'd rather address this if/when it happens, than making things difficult for us now. Here's a [Screenshot](https://screenshot.googleplex.com/pRHvitY3YXy) of the new UI. Clicking the ID column will correctly do a numeric sort).

Please let me know which of the above you disagree with.